### ISSUE TITLE
Spicetify stuck with solid color background

### ISSUE THREAD

**Issue by Narfee** [khanhas/spicetify-cli#615]
I've had spicetify installed on my system in the past, but I stopped using it and uninstalled it long ago after I stopped using spotify. 

After coming back to spotify I tried to get spicetify working again today, but nothing that I try gets it to load completely. Whenever I set a theme, spotify always loads up but just with the background color of the theme and the window bar elements. I've tried rebooting my system, uninstalling spotify, switching spicetify versions, spicetify backups, and nothing that I do lets me get spicetify to actually load the theme. 

I am running windows 10, using scoop to install spicetify-cli. 

Let me know if you know of a possible fix, I'd really love to get this great app running again!

![image](https://user-images.githubusercontent.com/24984088/118696901-60ce1380-b7d4-11eb-9181-3523d4827adb.png)

**Comment by Narfee** [comment]
Ok, so I deleted everything and reinstalled it and made sure I had v2.2.1, which I do, so that I could use spicetify for the new update to spotify. I realized that some of the themes could be the issue so I tried to apply the SpicetifyDefault theme. However, now I just get this

![image](https://user-images.githubusercontent.com/24984088/118721005-e8754b80-b7ef-11eb-85b2-ad0fc67fbcbb.png)

and the issue I had earlier persists.

**Comment by nintendork07** [comment]
hopefully just a temporary fix, but you can restore the default spotify theme with the "./spicetify restore" command since you're getting a blank spotify screen like i was. at least it'll work then until the (amazing) developers can fix it.

**Comment by Narfee** [comment]
@nintendork07 I finally got it working! 

someone shared a link to a spotify 1.5.6 download in the discord server [https://discord.gg/AxbXEPBnu7]. I scanned it with Malwarebytes and Virus total to validate it and nothing showed up, but always be careful when downloading exe's, it's at your own risk. 

Anyway I uninstalled spicetify and reinstalled it with the scoop method and it worked! 

hopefully spotify won't auto update so it'll stay working...
